homeassistant/components/flume/coordinator.py

```diff
@@ -109,9 +111,13 @@ def _update_lists(self):
                 continue
             device_id = notification["device_id"]
             rule = notification["extra"]["event_rule_name"]
-            active_notifications_by_device.setdefault(device_id, set()).add(rule)
+            time = parse_datetime(notification["created_datetime"])
+            age = -1 * time
```

Comment:
Is the `-1` needed?

Reply:
Youa re 100% correct...

> 2022-10-29 14:29:52.285 ERROR (MainThread) [homeassistant.components.flume] Error fetching flume data: Error communicating with flume API: unsupported operand type(s) for *: 'int' and 'datetime.datetime'

Reply:
Well it seems to work ok with just `time`

🤷🏼